pkg/crawl/identify.go

```diff
@@ -313,6 +320,28 @@ func (i *Log4jIdentifier) lookForMatchInTar(ctx context.Context, getTarReader ar
 	return archiveResult, versions, nil
 }
 
+func (i *Log4jIdentifier) printDetailedHashFinding(path string, finding Finding) {
+	if finding&ClassFileMd5 > 0 {
+		i.printInfoFinding("Found JndiManager class that was an exact md5 match for a known version at %s", path)
```

Comment:
Don't think this outputs what you want -- as currently written, I think this will output a literal `%s` since `printInfoFinding` takes `message` and `location` but prints both literally.

Either `printInfoFinding` should take a format string or the calls should render directly.

Reply:
I don't think this is the case as the colour library takes formatted strings and outputs e.g.:
```
[INFO] Found finding in what appeared to be an obfuscated jar at examples/obfuscated/2.14.1-aaaagb.jar
```